http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriberSpec.scala

```diff
@@ -157,6 +157,23 @@ class LeoPubsubMessageSubscriberSpec
     }
   }
 
+  // TODO
```

Comment:
shall we add these todo?

Reply:
Yeah, I was struggling with how to test those because they just call `RuntimeService` methods and return `IO[Unit]`. Feels like I would need to set up mocks and capture the calls to them somehow.

Reply:
I think we can also just check some logic branching is working as expected...like for `handleDeleteRuntimeMessage`, we can check the `ClusterInvalidState` path at least

Reply:
sounds good, will add